ethereum/core/src/test/java/org/hyperledger/besu/ethereum/core/encoding/TransactionRLPDecoderTest.java

```diff
@@ -32,7 +32,7 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
-class TransactionDecoderTest {
+class TransactionRLPDecoderTest {
```

Comment:
Is this "as opposed to SSZ"? Should we update the production class names as well to match?

Not for this PR, but maybe the other Encoders/Decoders should be renamed to